.github/workflows/ci_controller.yml

```diff
@@ -0,0 +1,29 @@
+name: ci:controller
+
+on:
+  push:
+    branches: master
+  pull_request:
+    paths:
+    - '*'
+    - 'packages/controller/**'
```

Comment:
Isn't this overridden by the `'*'` above?

Reply:
No, the * only selects the root dir. Although in the case of this repo, as it doesn't use lerna or yarn workspaces, it technically doesn't matter.